### DISCUSSION TITLE
Native Chatbot support

### DISCUSSION THREAD

**Discussion by jannick-ux** [SchwarzIT/onyx#884] [Ideas]
onyx will be modular, so I think it could be nice to give the opportunity to easily integrate a chatbot into the system. Visually it could recycle already concepted modules like the notificationcenter for example. But other integrations could be possible as well. To sum it up: Chatbot as an additional optional module for onyx

**Comment by mj-hof** [comment]
Probably a new component or feature that needs UX concept with a general approach. The logic for that chatbot should be defined in the projects.

**Comment by DeadlyVirez** [comment]
[Here](https://www.agent-native.com/docs/what-is-agent-native) is some inspiration for that

**Comment by larsrickert** [comment]
We currently do not plan specific components for this but we are open to re-discuss this in the future if there are more feature requests for this.
